src/controllers/user.controller.js

```diff
@@ -10,7 +10,11 @@ export class UserController {
 			const { id, email } = await UserService.register(user);
 			const userData = { id, email };
 			const userToken = JwtUtility.generateToken(userData);
-			return res.status(201).json({ user: userData, token: userToken });
+      UserService.sendVerificationEmail(user, userToken);
+			return res.status(201).json({message: 'Check your email to confirm your account, if not received do register again', 
```

Comment:
- This message is wrong, can you explain what will happen if the user registered again?
- Do you think the only way to get verified is to register again?

Reply:
I have made the change, user will not need to register again instead will hint /api/v1/users/resend-verify-email to resend confirmation email if did not received after being registered

> * This message is wrong, can you explain what will happen if the user registered again?
> * Do you think the only way to get verified is to register again?